src/getCoinCombination.test.js

```diff
@@ -7,4 +7,59 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('combination of coins for 1 cent', () => {
+    const result = getCoinCombination(1);
+
+    expect(result).toEqual([1, 0, 0, 0]);
```

Comment:
Also a small suggestion how to make it shorter.
```suggestion
    expect(getCoinCombination(1))
      .toEqual([1, 0, 0, 0]);
```

Reply:
fixed